### PR TITLE
update timespan.xml with new PWSH sample

### PR DESCRIPTION
**Pull request by doctordns**
# Update timespan.xml with new PWSH sample

## Summary
Added a new sample
Updated timespan1.xml

## Details

This is a change done off line via git.

## Suggested Reviewers

If you know who should review this, use '@' to request a review.
@rpetrusha 
@mairaw

### CODE REVIEW

**Review by rpetrusha — CHANGES_REQUESTED** [review]
There's only one file (the PowerShell example) in your PR, @doctordns. So this example won't appear anywhere in the TimeSpan documentation unless you also include TimeSpan.xml.

File: samples/snippets/powershell/VS_Snippets_CLR_System/ps/structure1.ps1 [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# <snippet1>
```
Comment (rpetrusha) [inline samples/snippets/powershell/VS_Snippets_CLR_System/ps/structure1.ps1]: Since there's only one example here, do you need the snippet tags?

Comment (doctordns) [reply samples/snippets/powershell/VS_Snippets_CLR_System/ps/structure1.ps1]: as far as the snippet tags - I thought these were needed to match up with the "#1" in the XML.

**Comment by mairaw** [comment]
CLA was not working. Closed and reopened to get the status changed.

**Comment by jeffwilcox** [comment]
@mairaw looks like the status is good now!

**Comment by mairaw** [comment]
@jeffwilcox yes! CLA is working again.

File: samples/snippets/powershell/VS_Snippets_CLR_System/ps/structure1.ps1 (hunk 1 above)
Comment (rpetrusha) [reply samples/snippets/powershell/VS_Snippets_CLR_System/ps/structure1.ps1]: @doctordns, use snippet tags if you have multiple examples in a single file. If the example is the entire file, don't use snippet tags in the file, and don't include the snippet reference after the filename and file path.

File: xml/System/TimeSpan.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <TypeSignature Language="C#" Value="public struct TimeSpan : IComparable, IComparable&lt;TimeSpan&gt;, IEquatable&lt;TimeSpan&gt;, IFormattable" />
   <TypeSignature Language="ILAsm" Value=".class public sequential ansi serializable sealed beforefieldinit TimeSpan extends System.ValueType implements class System.IComparable, class System.IComparable`1&lt;valuetype System.TimeSpan&gt;, class System.IEquatable`1&lt;valuetype System.TimeSpan&gt;, class System.IFormattable" />
   <TypeSignature Language="DocId" Value="T:System.TimeSpan" />
-  <AssemblyInfo>
+  <AssemblyInfo>cs/structure1.cs#1)]
```
Comment (rpetrusha) [inline xml/System/TimeSpan.xml]: Please remove `cs/structure1.cs#1)]`.

File: xml/System/TimeSpan.xml [hunk 3]
```diff
@@ -128,7 +128,8 @@
  The following example instantiates a <xref:System.TimeSpan> object that represents the difference between two dates. It then displays the <xref:System.TimeSpan> object's properties.  
   
  [!code-csharp[System.TimeSpan.Class#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.timespan.class/cs/structure1.cs#1)]
- [!code-vb[System.TimeSpan.Class#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.timespan.class/vb/structure1.vb#1)]  
+ [!code-vb[System.TimeSpan.Class#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.timespan.class/vb/structure1.vb#1)]
+ [!code-powershell[System.Timespan.Class#1](~/samples/snippetspowershell/VS_Snippets_CLR_System/system.timespan.class/ps/structure1.ps1]
```
Comment (rpetrusha) [inline xml/System/TimeSpan.xml]: The path is incorrect -- it's `~/samples/snippets/powershell
Also, if you're not going to use a snippet reference, you should remove the begin snippet and end snippet tags from the PowerShell example.

**Review by rpetrusha — COMMENTED** [review]
The sample isn't building, @doctordns. I've left a couple of comments.

File: samples/snippets/powershell/VS_Snippets_CLR_System/ps/structure1.ps1 (hunk 1 above)
Comment (doctordns) [reply samples/snippets/powershell/VS_Snippets_CLR_System/ps/structure1.ps1]: Right. I've remove the snippet tag, removed that weird additioin to XML and I think put the snippet in the right place. Have pushed this all up and let's see what happens.

I will eventually get this all figured out at my end! ;-)

File: xml/System/TimeSpan.xml [hunk 4]
```diff
@@ -128,7 +128,12 @@
  The following example instantiates a <xref:System.TimeSpan> object that represents the difference between two dates. It then displays the <xref:System.TimeSpan> object's properties.  
   
  [!code-csharp[System.TimeSpan.Class#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.timespan.class/cs/structure1.cs#1)]
- [!code-vb[System.TimeSpan.Class#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.timespan.class/vb/structure1.vb#1)]  
+ [!code-vb[System.TimeSpan.Class#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.timespan.class/vb/structure1.vb#1)]
+<<<<<<< HEAD
+ [!code-powershell[System.Timespan.Class#1](~/samples/snippets/powershell/VS_Snippets_CLR_System/system.timespan.class/ps/structure1.ps1]  
+=======
+ [!code-powershell[System.Timespan.Class#1](~/samples/snippetspowershell/VS_Snippets_CLR_System/system.timespan.class/ps/structure1.ps1]
+>>>>>>> 82e294c5709be80c8c994ce2bb7d90cc02f6860d
```
Comment (rpetrusha) [inline xml/System/TimeSpan.xml]: @doctordns, this is a merge conflict that GitHub wanted you to resolve; it's marked off the two variants of the text. To resolve it, you determine which version of the text is the correct one (in this case, the first is). Then delete `<<<<<<< HEAD `, and delete the older variant as well as the GitHub tags:
```
======= 
[!code-powershell[System.Timespan.Class#1](~/samples/snippetspowershell/VS_Snippets_CLR_System/system.timespan.class/ps/structure1.ps1] 
>>>>>>> 82e294c5709be80c8c994ce2bb7d90cc02f6860d 
```

**Comment by rpetrusha** [comment]
@doctordns, I left a comment about resolving a merge conflict in the text. Also, in an earlier commit, you'd inadvertently added some text after an \<AssemblyInfo> element; that needs to be removed, since it potentially could cause a build failure.

**Comment by doctordns** [comment]
I have been off ill, and now need to travel to Germany for a couple of weeks. 
This PR is in a bit of a messy state (my own mistakes, I'm afraid). So I'm closing it, and when I get back to the UK I'll try again.

**Comment by rpetrusha** [comment]
I hope that you're fully recovered and have a good trip, @doctordns.

**Comment by doctordns** [comment]
After a couple of weeks teaching the USAF a bit about PowerShell, I'm back! And I issued a separate P/R to add this change into the documents. Just waiting for it to get approved!

**Pull request closed without merging**